lumicks/pylake/scan.py

```diff
@@ -43,18 +42,32 @@ def num_frames(self):
 
     @property
     def lines_per_frame(self):
-        return self._get_axis_metadata(1)["num of pixels"]
+        return self.json["scan volume"]["scan axes"][1]["num of pixels"]
+
+    def _flip_axis(self, data):
```

Comment:
Maybe this could be called `_flip_if_needed` or something else that implies that the flip only happens sometimes.

Or you could also go with something like `_to_spatial` as you could then explain that the reconstruction returns a `[slow, fast]` structure and we want to return a `[Y, X]` one.

Reply:
`to_spatial` sounds good to me. I improved the docstring a bit as well.